tests/end_to_end_tests/conftest.py

```diff
@@ -13,7 +13,10 @@
 
 @pytest.fixture(scope="function")
 def first_grayskull_device():
-    return tt_lib.device.CreateDevice(0)
+    device = tt_lib.device.CreateDevice(0)
+    yield device
+
+    tt_lib.device.CloseDevice(device)
```

Comment:
Lol clearly I'm a fucking BRICK

Reply:
No need for the `DeallocateBuffers` etc ?

Reply:
No, in the interest of keeping things minimal I didn't pull that logic here. The end to end tests don't use Buffers

Reply:
what's the best way to keep an eye on whether resources get split up like buffers like this?
Just check root conftest fixture?

Reply:
Yea.. I like the concept of keeping the testing here to a minimal. Just a refactor/test updating mistake.